pulumi/backuprunner/src/scaling.ts

```diff
@@ -0,0 +1,57 @@
+import k8s from "@kubernetes/client-node";
+
+export const delay = (time: number) => {
+  return new Promise((resolve) => setTimeout(resolve, time));
+};
+
+export const scaleStatefulSet = async (
+  k8sAppsClient: k8s.AppsV1Api,
+  name: string,
+  namespace: string,
+  count: number,
+  waitToFinish: boolean
+) => {
+  console.log(`Scaling StatefulSet ${namespace}.${name} to ${count} replicas`);
+
+  var res = await k8sAppsClient.readNamespacedStatefulSet(name, namespace);
```

Comment:
can also destructure
```suggestion
  const { body } = await k8sAppsClient.readNamespacedStatefulSet(name, namespace);
```